config/routes.rb

```diff
@@ -0,0 +1,17 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  root 'home#index'
+
+
+  resources :trips
+  resources :passengers
+   resources :passengers, except: :destroy  do
+    resources :trips, only: :create
+    member do
+      get :trips, to: 'passengers#index'
```

Comment:
Interesting! My understanding of this section of code is that it gives you a named path named `trips_passenger_path` for a GET at `/passengers/:id/trips` that will go to the PassengersController index action specifically. However, I don't see any use of the `trips_passenger_path` in this project... What did you use this member block for that I don't see?